### DISCUSSION TITLE
Keep track on old issues and close.

### DISCUSSION THREAD

**Discussion by JSuenram** [Ylianst/MeshCentral#3637] [Ideas]
I started to request confirmation from users if old Issues are solved and asked them to close if so.
Hope this helps to clean up the Issue-List a little bit.
I can also confirm that many older Issue are solved, but people never report back.

I think we need a solution for this. 900+ Issues is far to many for few contributors to keep track of.
Maybe we need someone who is also allowed to close issues when people never come back.

I am not sure if a bot is a right solution, cause it can not confirm if a Issue is solved. Only close older ones by date....

**Comment by Ylianst** [comment]
Not a high priority for me, but yes, I need to first deploy a bot to deal with this before we do anything manually. I would like a bot with a predictable behavior so everyone knows what to expect.
